components/skeleton/test/skeleton.vdiff.js

```diff
@@ -0,0 +1,65 @@
+import '../demo/skeleton-test-box.js';
+import '../demo/skeleton-test-container.js';
+import '../demo/skeleton-test-heading.js';
+import '../demo/skeleton-test-link.js';
+import '../demo/skeleton-test-paragraph.js';
+import '../demo/skeleton-test-width.js';
+import '../demo/skeleton-test-stack.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+describe('d2l-skeleton', () => {
+
+	['ltr', 'rtl'].forEach(dir => {
+
+		const rtl = dir === 'rtl';
+
+		describe(dir, () => {
+
+			['standard', 'compact', 'small', 'label'].forEach((type) => {
```

Comment:
This will add 3 more "lines" tests for the "label" type, but seemed worth it to get the nested grouping.